test/e2e/pod_stats_test.go

```diff
@@ -17,8 +17,9 @@ var _ = Describe("Podman pod stats", func() {
 	)
 
 	BeforeEach(func() {
-		if os.Geteuid() != 0 {
-			SkipIfCgroupV2("--cgroup-manager=cgroupfs which doesn't work in rootless mode")
+		SkipIfRootless("Tests fail with both CGv1/2 + required --cgroup-manager=cgroupfs")
+		if isContainerized() {
+			SkipIfCgroupV1("All tests fail Error: unable to load cgroup at ...: cgroup deleted")
```

Comment:
A better way to word these messages is `Skip...("FIXME: #12345: Tests failing due to xyz")`, where 12345 is a PR you've filed showing an exact reproducer along with its symptoms. This has many benefits: first and foremost, it tracks the problem. Second, once the issue is fixed, it is easy for the developer to grep for the issue number and re-enable the test. Finally, it makes it immediately obvious to a maintainer that the given `Skip` is a real problem, not a shrug-unsupported like the ones above. 

This is not worth blocking on, but since you have to resubmit the PR anyway once #8726 is fixed, I'd encourage you to address this one.

Reply:
It's a good suggestion, I'll keep that in mind.